go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -136,7 +136,7 @@ func updateDBRetryCounter(retryCounter *int32, db *util.OvsDbProperties) {
 		*retryCounter = 0
 	} else {
 		*retryCounter += 1
-		klog.Infof("Failed to get cluster status for: %s, number of retries: %d", db, *retryCounter)
+		klog.Infof("Failed to get cluster status for: %v, number of retries: %d", db, *retryCounter)
```

Comment:
nit: `Failed to get cluster status for %v after %d retries`

Reply:
yup done. much better for understanding.